doc/ReleaseNotes.md

```diff
@@ -3,6 +3,11 @@ The release versions are NuGet releases.
 
 ## Unreleased (master)
 
+### Fixes
+* fixed filled polyline not displayed with stroke-width=0 (see [#785](https://github.com/svg-net/SVG/issues/785)
```

Comment:
append ` and PR #816`.

Reply:
The issue is linked with the PR anyway - I usually link with the PR only if there is no issue for it. This is the way it is done in other projects, too.